### DISCUSSION TITLE
Is it on purpose that warnings have a smaller font size?

### DISCUSSION THREAD

**Discussion by C0DK** [pradyunsg/furo#138] [Potential Issue] [answered]
It's actually right in the example photo, where you see a warning box:

![image](https://user-images.githubusercontent.com/9048078/116932994-42f4a080-ac63-11eb-8d89-752d6fa6a009.png)

But in my opinion, it doesn't look super good that the warning box text is smaller than the body text:
![image](https://user-images.githubusercontent.com/9048078/116933098-5e5fab80-ac63-11eb-8d1b-a899d02e557d.png)

I'd love to either customize it or it to be different by design. Would it be difficult to change? I have no experience with Furo themes.

**Comment by C0DK** [comment]
So one can customly set these in the Sphinx config:

```python
html_theme_options = {
    "light_css_variables": {
        "admonition-title-font-size": "1.25rem",
        "admonition-font-size": "1rem",
    },
    "dark_css_variables": {
        "admonition-title-font-size": "1.25rem",
        "admonition-font-size": "1rem",
    },
}
```
![image](https://user-images.githubusercontent.com/9048078/116937339-602c6d80-ac69-11eb-9eac-e17a560bf987.png)

Alternatively, if you like this version better, it is simply a matter of setting the same values in 
https://github.com/pradyunsg/furo/blob/be3efbd3649db0b261dca781d43717442663df51/src/furo/assets/styles/variables/_index.scss

**Comment by pradyunsg** [comment, marked as answer]
As noted already, these are configurable. The choice to make them smaller is a stylistic one and has been discussed before.

From https://github.com/pradyunsg/furo/issues/104#issuecomment-803266495:

> It's essentially an opinionated design choice, and there's a knob for it too -- admonitions are supposed to be side content, something that's an aside from the regular text in the document. At the same size as the rest of the page, they look significantly more eye catching and distract the content's flow way more than I'd like.
>
> I looked around and there seemed to be an even split in how they're handled, and I picked the approach that mkdocs-material takes on that front; since that's where I borrowed the admonitions' design from. :)

If you care enough to feel like you want it to be different, I imagine you'd also be fine with setting values in the configuration file.

**Reply by OriolAbril** [reply]
Hi, I hope you don't mind posting on the old topic. I too was surprised by the smaller font for admonitions, now I am wondering if I have been using admonitions "wrong" (aka not the way they were intended too). Without having read anything about admonitions, I assumed their intended goal was to emphasize key content, not for side content. Could you share some of the references that got you to this interpretation?

> admonitions are supposed to be side content, something that's an aside from the regular text in the document. 

The docs for admonitions in both [sphinx](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#directives) and [docutils](https://docutils.sourceforge.io/docs/ref/rst/directives.html#admonitions) are quite vague, but the [cpython devdocs](https://devguide.python.org/documentation/markup/#paragraph-level-markup) seem to indicate otherwise. I am quite comfortable using sphinx, but I don't know much about technical writing and I know nothing about web or visual design.

I am not suggesting to change the current style, much less trying to change your opinion. I would like to learn how to style the docs I write so users intuitively understand what content is an aside, what are key takeaways...

Thanks
